ani-cli

```diff
@@ -444,14 +443,16 @@ shift $((OPTIND - 1))
 
 case $scrape in
 	query)
+        base_url=$(curl -s -L -o /dev/null -w "%{url_effective}\n" https://gogoanime.cm)
 		get_search_query "$*"
 		search_results=$(search_anime "$query")
 		[ -z "$search_results" ] && die "No search results found"
 		anime_selection "$search_results"
 		episode_selection
 		;;
 	history)
-		search_results=$(sed -n -E 's/\t[0-9]*//p' "$logfile")
+        base_url=$(curl -s -L -o /dev/null -w "%{url_effective}\n" https://gogoanime.cm)
```

Comment:
Please move the base_url assignment before the case statement

Reply:
Did everything